### ISSUE TITLE
Search for learning rate scheduler

### ISSUE THREAD

**Issue by haifeng-jin** [keras-team/autokeras#897]
### Feature Description

Enlarge the search space for learning rate scheduler for the optimizer

### Solution

Use the learning rate schedulers here:
https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/schedules/ExponentialDecay

**Comment by fchollet** [comment]
Do we ever expect users to want to specify losses and metrics *otherwise* than through the Head objects?

We could add a `compile` step with only the `optimizer` argument.

**Comment by haifeng-jin** [comment]
We will not let the user specify the optimizer for now and just search for the best optimizer by enlarging the search space.
